PWGGA/GammaConv/macros/AddTask_GammaConvCalo_pp.C

```diff
@@ -2301,12 +2308,12 @@ void AddTask_GammaConvCalo_pp(
     cuts.AddCutPCMCalo("0008d113", "0dm00009f9730000dge0404000","4117937067032220000", "0163103100b00010"); // NonLin 37
 
   } else if (trainConfig == 2020){ // EMCAL+DCAL clusters standard cuts, INT7, NL , std TM, tight timing
-    cuts.AddCutPCMCalo("00010113","0dm00009f9730000dge0404000","4117900067032230000","0163103100b00010"); // INT7 NoNL
-    cuts.AddCutPCMCalo("00010113","0dm00009f9730000dge0404000","4117901067032230000","0163103100b00010"); // INT7 TBNL
-    cuts.AddCutPCMCalo("00010113","0dm00009f9730000dge0404000","4117911067032230000","0163103100b00010"); // INT7 NL11
-    cuts.AddCutPCMCalo("00010113","0dm00009f9730000dge0404000","4117912067032230000","0163103100b00010"); // INT7 NL12
-    cuts.AddCutPCMCalo("00010113","0dm00009f9730000dge0404000","4117921067032230000","0163103100b00010"); // INT7 NL21
-    cuts.AddCutPCMCalo("00010113","0dm00009f9730000dge0404000","4117922067032230000","0163103100b00010"); // INT7 NL22
+    cuts.AddCutPCMCalo("00010113","0dm00009f9730000dge0404000","4117900067032220000","0163103100b00010"); // INT7 NoNL
```

Comment:
what is the reason for the M02 cut change?

Reply:
Actually it was an mistake that it was 3 before. I just set it to my default value 2 (M02=0.7) for all variations

Reply:
Committed a change